Grievance_Backend/grievance/src/main/java/com/grievance/controller/EmployeeController.java

```diff
@@ -115,22 +121,23 @@ public ResponseEntity<?> listAllEmployees(
    */
   @PostMapping("/saveEmployee")
   public ResponseEntity<?> saveEmployee(
-          @RequestParam final String email,
-          @RequestParam final String password,
+          @RequestHeader final String email,
+          @RequestHeader final String password,
           @RequestBody final EmployeeInDto employeeInDto) {
 
-     Boolean boolean1 =
-     authenticatingUser.checkIfUserIsAdmin(email, password);
-     if (boolean1) {
-       Optional<EmployeeOutDto> optional =
-             employeeService.saveEmployee(employeeInDto);
-               if  (!optional.isPresent()) {
-                     throw new EmployeeAlreadyExistException(
-                       employeeInDto.getEmail());
-               }
-               return new ResponseEntity<>(optional, HttpStatus.ACCEPTED);
-            }
-       return new ResponseEntity<>("Invalid Authorization",
-           HttpStatus.UNAUTHORIZED);
+     try {
+          authenticatingUser.checkIfUserIsAdmin(email, password);
```

Comment:
our checkIfUserIsAdmin method is returning boolean (user is admin or not), you are not assiging it to variable also not ccomparing that value anywhere.